content/terms.md

```diff
@@ -156,7 +156,7 @@ Fees are exclusive of taxes. Customer is responsible for the payment of all taxe
 # Term; Termination
 
 ## Term
-Unless otherwise set forth in the applicable Purchase Order, such Purchase Order will have an Initial Term of three (3) years starting on the first day of the Initial Term set forth in the applicable Purchase Order (“Effective Date” or “Start Date”). Unless otherwise set forth in the applicable Purchase Order, all Purchase Orders for the Subscription Service will thereafter automatically renew for the same period of time as the initial Subscription Term (each a “Renewal Term”) unless either party notifies the other party at least ninety (90) days prior to the then current expiration date of the Initial Term or the Renewal Term, as the case may be, that it has elected not to renew the Purchase Order. Akeneo reserves the right to change the rates, applicable charges and usage policies and to introduce new charges, for such Purchase Order upon providing Customer with written notice thereof (which notice may be provided by e-mail) at least 60 days prior to the then current renewal date of the Purchase Order.
+Unless otherwise set forth in the applicable Purchase Order, such Purchase Order will have an Initial Term of three (3) years starting on the first day of the Initial Term set forth in the applicable Purchase Order (“Effective Date” or “Start Date”). Unless otherwise set forth in the applicable Purchase Order, all Purchase Orders for the Subscription Service will thereafter automatically renew for the same period of time as the initial Subscription Term (each a “Renewal Term”) unless either party notifies the other party at least sixty (60) days prior to the then current expiration date of the Initial Term or the Renewal Term, as the case may be, that it has elected not to renew the Purchase Order. Akeneo reserves the right to change the rates, applicable charges and usage policies and to introduce new charges, for such Purchase Order upon providing Customer with written notice thereof (which notice may be provided by e-mail) at least ninety (90) days prior to the then current renewal date of the Purchase Order.
```

Comment:
```suggestion
Unless otherwise set forth in the applicable Purchase Order, such Purchase Order will have an Initial Term of three (3) years starting on the first day of the Initial Term set forth in the applicable Purchase Order (“Effective Date” or “Start Date”). Unless otherwise set forth in the applicable Purchase Order, prior to the end of the Initial Term, Parties agree to enter into renewal negotiations ninety (90) days from the anniversary date. If the Parties fail to enter into a renewal agreement, the Agreement will automatically expire unless explicitly extended by an instrument in writing signed by both Parties. Akeneo reserves the right to change the rates, applicable charges and usage policies and to introduce new charges, for such Purchase Order upon providing Customer with written notice thereof (which notice may be provided by e-mail) at least ninety (90) days prior to the then current renewal date of the Purchase Order.
```